### PR TITLE
Add alarms for FreeBSD interface errors

### PR DESCRIPTION
**Pull request by lassebm**
##### Summary
This PR add alarms for FreeBSD interface errors, inbound and outbound, based on net.drops alarms.

##### Component Name
health.d

##### Description of testing that the developer performed
Used for weeks on multiple FreeBSD routers running Netdata 1.19.0.

### CODE REVIEW

**Comment by lassebm** [comment]
Any interest in reviewing this PR?

**Comment by thiagoftsm** [comment]
> Any interest in reviewing this PR?

Hi @lassebm ,

Firstly I apologize the delay, but we had a busy month.
I am testing your PR right now.

Best regards!

File: health/health.d/net.conf [hunk 1]
```diff
@@ -110,6 +110,34 @@ families: *
     info: the ratio of outbound dropped packets vs the total number of sent packets of the network interface, during the last 10 minutes
       to: sysadmin
 
+# -----------------------------------------------------------------------------
+# interface errors
+
+template: interface_inbound_errors
+      on: net.errors
```
Comment (thiagoftsm) [inline health/health.d/net.conf:117]: Hi @lassebm ,

Netdata applies templates only for `contexts` and we do not have any context named `net.errors`. To find a `context` name you can do the following request `curl -o charts.txt http://localhost:19999/api/v1/charts?all`.

For these two alarms that you are adding, you should use `ipv4.errors` that is the context for ipv4 error chart.

Best regards!

Comment (lassebm) [reply health/health.d/net.conf:117]: The added alarms are for interface errors, not IPv4. See https://github.com/netdata/netdata/blob/master/collectors/freebsd.plugin/freebsd_getifaddrs.c#L525.

Comment (thiagoftsm) [reply health/health.d/net.conf:117]: I am sorry for the confusion, my laptop does not have these charts, so I confused with traffic errors :(

**Pull request merged**